pkg/core/portal.go

```diff
@@ -299,6 +301,7 @@ func (p *AuthPortal) ServeHTTP(w http.ResponseWriter, r *http.Request, upstreamO
 			}
 
 			claims := resp["claims"].(*jwtclaims.UserClaims)
+			p.supplementClaims(claims)
```

Comment:
@SvenDowideit , there are the rules for adding roles to an email. However, there could be role stripping rules in the future. In `oauth2` backend I called it `supplementClaims()`. In retrospect, I think it needs to be changed to accomodate the about.

Reply:
changed to ApplyRoleMapToClaims - I started trying out the code for role stripping, but that brings up questions of what order does one apply the mappings in - and tbh, I don't need it, so will leave it for a future PR.